configure.ac

```diff
@@ -9,8 +9,8 @@ AC_CONFIG_AUX_DIR([config])
 AC_SUBST([ac_aux_dir])
 AC_CONFIG_HEADERS([config.h])
 AC_CONFIG_MACRO_DIR([config])
-# 'AM_PROG_AR' was introduced in Automake 1.11.2
-AM_INIT_AUTOMAKE([1.11.2 subdir-objects])
+# Make us require Automake 1.11.1
+AM_INIT_AUTOMAKE([1.11.1 subdir-objects])
```

Comment:
This is just so we're pinned to the latest version possible that still works. I doubt it will actually break it for anyone, given how old it is.